beetsplug/audible.py

```diff
@@ -520,3 +552,73 @@ def write_book_description_and_narrator(self, items):
             narrator = item.composer
             with open(os.path.join(destination, b"reader.txt"), "w") as f:
                 f.write(narrator)
+
+    def before_choose_candidate_event(self, session, task):
+        return [
+            PromptChoice('r', 'Region switch', self.book_level_region_switch)
+        ]
+
+    def book_level_region_switch(self, session, task):
+        """Prompts the book level region value"""
+        available_region_codes = ', '.join(
+            (ui.colorize("added", reg) for reg in AUDIBLE_REGIONS)
+        )
+
+        # config level region code
+        current_config_region_code = self.config['region'].get()
+
+        # book level region code
+        book_region_code = get_item_region(task.items[0])
+        if book_region_code is None:
+            current_book_region_code = '--'
+            ui_current_config_region_code = ui.colorize("text", current_config_region_code)
```

Comment:
The [UI colors](https://beets.readthedocs.io/en/stable/reference/config.html#colors) I use are only available in the Beets version 2.0.0. Should I rewrite this and other parts involving UI color to make them compatible with older versions of Beets, such as 1.6.0?